progress/progress.go

```diff
@@ -64,7 +64,7 @@ func WithOutput(w io.Writer) Option {
 // When in verbose mode, the bar will print simple status update lines.
 func WithVerbose(v bool) Option {
 	return func(b *Bar) {
-		b.verbose = v
+		b.Verbose = v
 	}
 }
```

Comment:
Now that the progress bar follows the convetion of the Verbose flag being an exported member on the struct, it is perhaps confusing to also have as a WithX option.  May want to remove.